project/frontend/src/hoc/PageRouter/PageRouter.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Route, Switch, Redirect } from "react-router-dom";
+
+import LandingPage from "../../containers/LandingPage/LandingPage";
+import SelectCompetition from "../../containers/SelectCompetition/SelectCompetition";
+import Dashboard from "../../containers/Dashboard/Dashboard";
+import MySGonks from "../../containers/MySGonks/MySGonks";
+import Marketplace from "../../containers/Marketplace/Marketplace";
+import Competition from "../../containers/Competition/Competition";
+
+import { NO_COMPETITION } from "../../App";
```

Comment:
may as well just define that in this file now

Reply:
I figured maybe it's better to have it just defined in one place to keep it consistent? That way if we ever need to change NO_COMPETITION's value to something else for whatever reason, we only need to check one place, rather than having to change it in multiple places. WDYT?

Reply:
If we were using it throughout the codebase (or plan to use it in another component later on) it would be worthwhile to keep it in the top scoped file, however since the only use of it is in PageRouter I think we should keep it local to that.

Reply:
Disregard, just saw that we're using it in App.js now 🤦